src/compass/utils/schemas/cslc_s1.yaml

```diff
@@ -34,18 +34,18 @@ runconfig:
         # This section includes parameters to tweak the workflow
         processing:
            # Polarization to process. If not set, process all polarizations for that burst
-           polarization: any(list('HH', 'HV', 'VH', 'VV'), min=1, max=4), str(min=2, max=2), required=False)
-
+           polarization: any(list('HH', 'HV', 'VH', 'VV'), min=1, max=2, required=True)
```

Comment:
"polarization" should be optional in my opinion. 
We don't want users (or SDS) be required to choose polarization. By default we should always extract polarization from the filename and process them. I'm hesitant to say if we should process both or a single polarization by default. We have two choices:
1- the default will be to always process co-pol channel
2- add an extra parameter indicating if we want to process co-pol, cross-pol or dual-pol channels. 
Option 2 may make the schema a bit more complicated but has the advantage that the data system can easily switch between co-pol , cross-pol, dual-pol without the need for specifying the exact polarizations. 
If we go with option-2 the existing ```polarization``` parameter should have the highest priority to decide the polarizations but again it should be optional.

Reply:
I think the changes in c0683a2 should match what you described.